services/clojars/clojars-release.service.js

```diff
@@ -0,0 +1,70 @@
+'use strict'
+
+const Joi = require('joi')
+const { version: versionColor } = require('../color-formatters')
+const { BaseJsonService, NotFound, InvalidResponse } = require('..')
+
+const clojarsSchema = Joi.object({
+  // optional due to non-standard 'not found' condition
+  latest_release: Joi.string(),
+}).required()
+
+module.exports = class ClojarsRelease extends BaseJsonService {
+  async fetch({ clojar }) {
+    const url = `https://clojars.org/api/artifacts/${clojar}`
+    return this._requestJson({
+      url,
+      schema: clojarsSchema,
+    })
+  }
+
+  static render({ clojar, version }) {
+    return {
+      message: `[${clojar} "${version}"]`,
+      color: versionColor(version),
+    }
+  }
+
+  async handle({ clojar }) {
+    const json = await this.fetch({ clojar })
+
+    if (!json.latest_release) {
+
+      throw new InvalidResponse({ underlyingError: new Error('version is invalid or null') });
```

Comment:
Two suggestions here:

1) This is a minor nit but can you remove the linebreaks above and below this line?
2) 
```suggestion
      throw new InvalidResponse({ prettyMessage: 'version is invalid or null' })
```

Reply:
fixed!